src/Carbon/CarbonInterval.php

```diff
@@ -621,6 +621,30 @@ public function add(DateInterval $interval)
         return $this;
     }
 
+    /**
+     * Multiply current instance given number of times
+     *
+     * @param float $factor
+     *
+     * @return static
```

Comment:
`@return $this` (static is for same Class but possible different instance)

Reply:
Agree, but `add`, `weeksAndDays` and `__call` return `static` and not a single method returns `$this` so it would be inconsistent to change it only here. Wouldn't it?

Reply:
`@return $this` is used everywhere it's relevant in the Carbon class, if wrong in CarbonInterval, CarbonInterval should align on Carbon.

Reply:
Out of scope for this PR I guess :)